std/range/primitives.d

```diff
@@ -2003,12 +2003,13 @@ ElementType!R moveAt(R)(R r, size_t i)
 }
 
 /**
-Implements the range interface primitive $(D empty) for built-in
-arrays. Due to the fact that nonmember functions can be called with
-the first argument using the dot notation, $(D array.empty) is
-equivalent to $(D empty(array)).
+Implements the range interface primitive $(D empty) for types that
+obey $(LREF hasLength) property and for narrow strings. Due to the
+fact that nonmember functions can be called with the first argument
```

Comment:
FWIW spelling is non-member: https://en.oxforddictionaries.com/definition/non-member